HandyNotes_Achievements.lua

```diff
@@ -65,14 +65,70 @@ function HNA:HandyNotesCoordsNear(c, coord)
 end
 
 
+function SortByTrackedAchievement(nodes, enabled)

```

Comment:
can `enabled` be a global variable instead?  alternatively, the caller can:

    if condition then
        nodes = SortByTrackedAchievement(nodes)
    end

and simplify the `SortByTrackedAchevements()` signature.